app/root.module.css

```
@@ -0,0 +1,9 @@
+.root {
+  width: 50svw;
+  min-width: 20rem;
+  height: 100vh;
+  margin: 0 auto;
+  padding: 0.5rem;
+  border: solid;
+  border-width: 0 0.5px 0 0.5px;
```

```suggestion
  border-width: 0 0.5px;
```
上下と左右それぞれが同じ場合は省略できます.